tasks/12_title-case/title-case.js

```diff
@@ -0,0 +1,5 @@
+export function titleCase(str) {
+  return str.split(' ').map((el) => el.charAt(0)
+      .toUpperCase() + el.slice(1)
+      .toLowerCase()).join(' ');
```

Comment:
Adding comments or splitting this single line solution in the future would be nice in order to increase readability